client/src/components/signUpForm/ParentSignUp.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from "react";
+import { Input, Button } from "antd";
+import { PlusOutlined } from "@ant-design/icons";
+import AddChild from "./AddChild";
+import { userDataParentInterface } from "../../interfaces";
+
+const ParentSignUp: React.ElementType = ({
+  inputValue,
+  addEmailChildren,
+}: userDataParentInterface) => {
+  const regex = /^\w+([\\.-]?\w+)*@\w+([\\.-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
It will be good if you explained what the regex does in a comment for later times during the project.